site/docs/numbers/searchingNumbers.mdx

```diff
@@ -17,261 +17,481 @@ export const accountId = "{accountId}";
+{
+    Name = "Available Telephone Number order",
+    SiteId = "461",
+    ExistingTelephoneNumberOrderType = {
+        TelephoneNumberList = [] {
+            TelephoneNumber: "6672286710",
+            TelephoneNumber: "5756186700"
+        }
+    }
+});
+```
 
-Parameters:
+</TabItem>
 
-* localVanity : 867_0*
-* lca         : false
-* quantity    : 2
+<TabItem value="nodejs">
+
+```nodejs
+var order = {
+    name: "Available Telephone Number order",
+    siteId: 461,
+    existingTelephoneNumberOrderType: {
+        telephoneNumberList: [
+            {
+                telephoneNumber: "6672286710"
+            },
+            {
+                telephoneNumber: "5756186700"
+            }
+        ]
+    }
+};
+numbers.Order.create(order, callback);
+```
 
-> Request
+</TabItem>
 
-```http
-GET https://dashboard.bandwidth.com/api/accounts/{accountId}/availableNumbers?localVanity=867_0*&quantity=2&lca=false
-Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+<TabItem value="php">
+
+```php
+$order = $account->orders()->create([
+    "Name" => "Available Telephone Number order",
+    "SiteId" => "461",
+    "ExistingTelephoneNumberOrderType" => [
+        "TelephoneNumberList" => [
+            "TelephoneNumber" => [ "6672286710", "5756186700" ]
+        ]
+    ]
+]);
 ```
 
+</TabItem>
+
+</Tabs>
+
+And endpoint `../orders` should return a response like:
+
 > Response
 
 ```xml
-HTTP/1.1 200
-Content-type: application/xml
-
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<SearchResult>
-    <ResultCount>2</ResultCount>
-    <TelephoneNumberList>
-        <TelephoneNumber>6672286710</TelephoneNumber>
-        <TelephoneNumber>5756186700</TelephoneNumber>
-    </TelephoneNumberList>
-</SearchResult>
+<OrderResponse>
+    <Order>
+        <OrderCreateDate>2018-01-23T19:56:29.678Z</OrderCreateDate>
+        <BackOrderRequested>false</BackOrderRequested>
+        <id>47955555-67aa-4adb-8c0f-b6894e60c0dc</id>
+        <ExistingTelephoneNumberOrderType>
+            <TelephoneNumberList>
+                <TelephoneNumber>6672286710</TelephoneNumber>
+                <TelephoneNumber>5756186700</TelephoneNumber>
+            </TelephoneNumberList>
+        </ExistingTelephoneNumberOrderType>
+        <PartialAllowed>true</PartialAllowed>
+        <SiteId>461</SiteId>
+    </Order>
+    <OrderStatus>RECEIVED</OrderStatus>
+</OrderResponse>
+```
+
+## Fetch Order Information
+
+Phone number ordering in the Bandwidth Dashboard is asynchronous when creating an "order". The orders are then processed and the order status is updated asynchronously.  Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `<OrderStatus>`.
+Order processing times can vary and are not guaranteed, so bandwidth does __not__ recommend setting a timeout on waiting for an order to show either `COMPLETE` or `FAILED` status, but instead relying on a webhook from an orders subscription.
+
+## Poll for Order Status
```

Comment:
Can we remove the poll section? I don't know that it makes sense to include since we don't recommend this method.